packages/jest-util/src/globsToMatcher.ts

```diff
@@ -44,13 +44,13 @@ export default function globsToMatcher(globs: Array<Config.Glob>): Matcher {
 
   const matchers = globs.map(glob => {
     if (!globsToMatchersMap.has(glob)) {
-      // Matchers that are negated have different behavior than matchers that
-      // are not negated, so we need to store this information ahead of time.
-      const {negated} = micromatch.scan(glob, micromatchOptions);
+      const isMatch = picomatch(glob, picomatchOptions, true);
 
       const matcher = {
-        isMatch: micromatch.matcher(glob, micromatchOptions),
-        negated,
+        isMatch,
+        // Matchers that are negated have different behavior than matchers that
+        // are not negated, so we need to store this information ahead of time.
+        negated: isMatch.state.negated || !!isMatch.state.negatedExtglob,
```

Comment:
we could ignore this type error temporarily so we can land this and get a pre-release out and then come back to update the types later.

Or we could add a patch (using `yarn patch`) so the types are correct

Reply:
Okay I hope it gets merged soon. Is there a new release of jest imminent?

Reply:
Currently in pre-release 🙂

I was hoping to do a release during Easter, but I never found the time... We've still got a few things in the [milestone](https://github.com/facebook/jest/milestone/12), but they're mostly minor or stuff we can punt for the next major (we're keeping node v10 support in 27, but I wanna make a somewhat quick v28 where we drop it for that native ESM support).

tl;dr: hopefully a stable this month

Reply:
I added a patch for now, will send a PR once the other PR is merged.